### ISSUE TITLE
Source Zendesk support: normalization failed after upgrade from v0.40.30 to v0.40.32

### ISSUE THREAD

**Issue by Wolff-Lucas** [airbytehq/airbyte#22197]
## Environment
- **Airbyte version**:  v0.40.32
- **OS Version / Instance**: debian 11 bullseye / GCP e2
- **Deployment**: Docker
- **Source Connector and version**: zendesk support 0.2.21
- **Destination Connector and version**: bigquery 1.2.13
- **Step where error happened**: Upgrade Airbyte

## Current Behavior
After having updated airbyte in version 0.40.32 the normalization part in error with the following error:
[b735c753_23c5_4f81_a30c_edaa85ea2054_logs_713_txt (1).txt](https://github.com/airbytehq/airbyte/files/10555865/b735c753_23c5_4f81_a30c_edaa85ea2054_logs_713_txt.1.txt)

data_type.WELL_KNOWN_TYPE_VAR_NAME in option[data_type.REF_TYPE_VAR_NAME] for option in properties[data_type.ONE_OF_VAR_NAME]
KeyError: '$ref'## Expected Behavior

A downgrade to v0.40.30 fixed the problem

*Tell us what should happen.*

Sync should succeed 

## Steps to Reproduce
1.Upgrade airbyte to v0.40.32
2.Launch zendesk support sync

## Are you willing to submit a PR?

No

**Comment by caiopedroso** [comment]
Even after downgrading to version 0.40.30, a lot of normalization stopped working (Typeform, Hubspot, and Zendesk) with the same error pattern, just a different field in the "KeyError: "

**Comment by caiopedroso** [comment]
+1

The downgrade to v0.40.30 broke others' syncs (Hubspot, Typeforms), so I'm kinda stuck in the latest version without Zendesk sync.

**Comment by marcosmarxm** [comment]
@caiopedroso and @Wolff-Lucas did you test version 0.40.33?

**Comment by mattimoller** [comment]
@marcosmarxm I am having the same issue and can test on 0.40.33, but can't see that release in the Airbyte releases 🤔 Could you point me to it?

**Comment by caiopedroso** [comment]
same as @mattimoller , no version 40.33 here yet

**Comment by caiopedroso** [comment]
Hi @marcosmarxm, do you know if 0.40.33 be released with the fix or only after separating the platform and connectors?

**Comment by markns** [comment]
I'm getting the same/similar issue using the Airbyte source connector on `0.40.32`

```
2023-02-20 23:39:50 [42mnormalization[0m > Traceback (most recent call last):
2023-02-20 23:39:50 [42mnormalization[0m >   File "/usr/local/bin/transform-catalog", line 8, in <module>
2023-02-20 23:39:50 [42mnormalization[0m >     sys.exit(main())
2023-02-20 23:39:50 [42mnormalization[0m >   File "/usr/local/lib/python3.9/site-packages/normalization/transform_catalog/transform.py", line 104, in main
2023-02-20 23:39:50 [42mnormalization[0m >     TransformCatalog().run(args)
2023-02-20 23:39:50 [42mnormalization[0m >   File "/usr/local/lib/python3.9/site-packages/normalization/transform_catalog/transform.py", line 36, in run
2023-02-20 23:39:50 [42mnormalization[0m >     self.process_catalog()
2023-02-20 23:39:50 [42mnormalization[0m >   File "/usr/local/lib/python3.9/site-packages/normalization/transform_catalog/transform.py", line 64, in process_catalog
2023-02-20 23:39:50 [42mnormalization[0m >     processor.process(catalog_file=catalog_file, json_column_name=json_col, default_schema=schema)
2023-02-20 23:39:50 [42mnormalization[0m >   File "/usr/local/lib/python3.9/site-packages/normalization/transform_catalog/catalog_processor.py", line 64, in process
2023-02-20 23:39:50 [42mnormalization[0m >     stream_processor.collect_table_names()
2023-02-20 23:39:50 [42mnormalization[0m >   File "/usr/local/lib/python3.9/site-packages/normalization/transform_catalog/stream_processor.py", line 227, in collect_table_names
2023-02-20 23:39:50 [42mnormalization[0m >     for child in self.find_children_streams(self.from_table, column_names):
2023-02-20 23:39:50 [42mnormalization[0m >   File "/usr/local/lib/python3.9/site-packages/normalization/transform_catalog/stream_processor.py", line 361, in find_children_streams
2023-02-20 23:39:50 [42mnormalization[0m >     elif is_combining_node(properties[field]):
2023-02-20 23:39:50 [42mnormalization[0m >   File "/usr/local/lib/python3.9/site-packages/normalization/transform_catalog/utils.py", line 120, in is_combining_node
2023-02-20 23:39:50 [42mnormalization[0m >     if data_type.ONE_OF_VAR_NAME in properties and any(
2023-02-20 23:39:50 [42mnormalization[0m >   File "/usr/local/lib/python3.9/site-packages/normalization/transform_catalog/utils.py", line 121, in <genexpr>
2023-02-20 23:39:50 [42mnormalization[0m >     data_type.WELL_KNOWN_TYPE_VAR_NAME in option[data_type.REF_TYPE_VAR_NAME] for option in properties[data_type.ONE_OF_VAR_NAME]
2023-02-20 23:39:50 [42mnormalization[0m > KeyError: '$ref'
```

**Comment by markns** [comment]
But it's resolved using the unreleased(?) 0.40.33 images.

**Comment by marcosmarxm** [comment]
Normalization and custom dbt are deprecated features and will be removed soon from the project. For that reason I'm closing the issue as it won't be implemented anymore. Please use destination Dv2
